CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_policy(SET CMP0048 NEW)  # project_VERSION* variables populated from project(... VERSION x.x.x) string
+
+project(Libint
+        VERSION 1.2.0
+        LANGUAGES C CXX)
+set(Libint_AUTHORS      "Edward F. Valeev and Justin T. Fermann")
+set(Libint_DESCRIPTION  "Parallel implementation of the Effective Fragment Potential (EFP) method")
+set(Libint_URL          "https://libefp.github.io/")
```

Comment:
Goodness, I'm sorry I missed substituting those. I'm off to a conference and will be back Saturday. Then I'll fix these up and investigate the one Psi4 anomaly in the full test suite (a string formatting/encoding issue). Thanks for the review.